modules/ROOT/pages/6.5.1-release-notes.adoc

```diff
@@ -383,13 +377,22 @@ With this release, the **Advanced Typography** plugin now correctly disables the
 
 For information on the **Advanced Typography** plugin, see: xref:advanced-typography.adoc[Advanced Typography].
 
+==== Fixed adding unnecessary space after a comma and before a quote in several scenarios.
+//#TINY-9510
+
+In previous versions of {productname}, the issue of adding an unnecessary space after a comma and before a quote in several scenarios was caused by an incorrectly written Regular Expression rule in the https://github.com/typograf[Typograf] library used by the Advanced Typography plugin. This led to the appearance of spaces between commas and quotes.
+
+To fix this, {productname} updated the Regular Expression rule in the Typograf library. As a result, spaces between commas and quotes no longer appear.
+
+For information on the **Advanced Typography** plugin, see: xref:advanced-typography.adoc[Advanced Typography].
+
 === PowerPaste 6.2.0
 
-The {productname} 6.5 release includes an accompanying release of the **Power Paste** premium plugin.
+The {productname} 6.5.1 release includes an accompanying release of the **Power Paste** premium plugin.
 
 **PowerPaste** 6.2.0 includes one fix:
 
-=== Existing tables were not filled out when pasting tables from Microsoft Word or Microsoft Excel.
+==== Existing tables were not filled out when pasting tables from Microsoft Word or Microsoft Excel.
 //#TINY-9500
 
 Previously, when the PowerPaste plugin was active and a table was copied from Microsoft Word or Micrsoft Excel into a {productname} editor, the copied table pasted as a whole table into a single cell, rather than merging the content of the copied table into the cells of the existing table.
```

Comment:
Typo -> Micrsoft to Microsoft

Reply:
```suggestion
Previously, when the PowerPaste plugin was active and a table was copied from Microsoft Word or Microsoft Excel into a {productname} editor, the copied table pasted as a whole table into a single cell, rather than merging the content of the copied table into the cells of the existing table.
```